exercises/practice/pythagorean-triplet/.docs/instructions.md

```diff
@@ -16,9 +16,15 @@ a < b < c
 For example,
 
 ```text
-3**2 + 4**2 = 9 + 16 = 25 = 5**2.
+3² + 4² = 9 + 16 = 25 = 5².
 ```
 
+Implement the method `isPythagorean((Int, Int, Int)): Boolean` which checks the given triplet. 
+
+Then implement `pythagoreanTriplets(min: Int, max: Int): Seq[(Int, Int, Int)]` which should returns a sequence of all
+triplets whose sides are in the range `min` to `max` inclusive
+
+Finally implement `pythagoreanTripletsSum(n: Int): Seq[(Int, Int, Int)]`
```

Comment:
Editing this file unfortunately is discouraged. That is due to the contents of the file being synced from a shared exercise source, in this case: https://github.com/exercism/problem-specifications/blob/main/exercises/pythagorean-triplet/description.md

If you need to add track-specific instructions, see https://exercism.org/docs/building/tracks/practice-exercises#h-file-docs-instructions-append-md

Reply:
Ah, I see. I was a bit surprised that descriptions weren't centralized, but since the Scala docs differed...

Reverting. I hope the easier to read "3²" will be synced to the Scala repo after my commit is merged.

Reply:
You could do this yourself via configlet. First run `./bin/fetch-configlet`. Then do: `./bin/configlet sync --docs --exercise pythagorean-triplets --update --yes`. We can also do this later 🤷